docs/site/docs/protocol/evmengine/diversity.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_position: 6
+---
+
+# Client Diversity
+
+Omni's EVM Engine champions the principle of client diversity, allowing the integration and use of any Ethereum Virtual Machine (EVM) client within its ecosystem. This approach underscores Omni's commitment to flexibility, scalability, and interoperability, promoting a robust and resilient blockchain network.
+
+### Integration of EVM Clients
+
+Omni supports the seamless integration of various EVM clients, such as Geth, Besu, Erigon, and others, without requiring specialized modifications. This inclusivity enables the Omni network to benefit from the diverse set of features and optimizations offered by different clients, enhancing the overall ecosystem.
+
+```go
+// Example of client diversity support
+func NewKeeper(...) *Keeper {
+    // Keeper initialization with flexible client integration...
+}
+```
```

Comment:
remove this - not about the keeper really. Since we adhere to the engine api, and all evm client adhere to the engine api, any evm client can be used.